src/cross_section/include/quality.h

```diff
@@ -8,6 +8,15 @@ class Quality {
   static float circularEdgeLength_;
 
  public:
+  /** @name Defaults
```

Comment:
Let's move this file to utilities, since it's shared.

Reply:
That is where I tried to have it initially, but I was having troubling getting it to compile as expected. I didn't explore changing it up much, but I think the problem might have stemmed from the utilities library being an interface, so the headers are included differently than usual. 

Do you have an idea of what I was doing wrong, or what would need to change to move the quality source and header over without issue?

Reply:
Actually, it would be nice to condense it into just the header. Want to try one of the solutions outlined [here](https://stackoverflow.com/questions/18860895/how-to-initialize-static-members-in-the-header)?

Reply:
I'll give it a shot during my next session 👍

Reply:
I've moved it into `public.h`, the `inline static` variable declaration worked. My LSP insists on warning me that it is a C++17 extension, but I added a definition to the utilities CMakeLists to suppress the warning for compiling purposes.